keel-core/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/DebianVersioningStrategy.kt

```diff
@@ -0,0 +1,20 @@
+package com.netflix.spinnaker.keel.artifacts
+
+import com.netflix.spinnaker.keel.api.artifacts.VersioningStrategy
+
+/**
+ * A [VersioningStrategy] that for Debian packages..
+ */
+object DebianVersioningStrategy : VersioningStrategy {
+  override val type: String = "debian"
```

Comment:
The fact that this has a different name than the version strategy we were using previously shouldn't have any impact as it's not present in delivery configs or persisted to the database.